### PR TITLE
chore: delete modules and use latest from npm

### PR DESCRIPTION
**Pull request by eladb**
Deletes the code of everything under `packages/**` and takes a dependency on the published modules instead. Modules are new mastered in [cdk8s-core], [cdk8s-cli] and [cdk8s-plus-17].

[cdk8s-core]: https://github.com/cdk8s-team/cdk8s-core
[cdk8s-cli]: https://github.com/cdk8s-team/cdk8s-cli
[cdk8s-plus-17]: https://github.com/cdk8s-team/cdk8s-plus-17

Doc website, examples and integration tests are now executed against the released versions (installed via the root `package.json` file).

---

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: Build
-on: [pull_request]
+on: [push]
```
Comment (iliapolo) [inline .github/workflows/build.yml:2]: Why the change?

File: examples/java/crd/package.json [hunk 2]
```diff
@@ -7,12 +7,5 @@
     "build": "cdk8s import --language java",
     "synth": "mvn install && mvn compile && cdk8s synth",
     "test": "echo 'no test for java yet.'"
-  },
-  "dependencies": {
```
Comment (iliapolo) [inline examples/java/crd/package.json:11]: Can't we leave these with the current latest version and let the dependency upgrade workflow take care of upgrading it? (i'm not sure why it didn't already...)

**Review by iliapolo — COMMENTED** [review]
@eladb Maybe add a README note for a while saying the packages were moved? I would also post something on our slack channel notifying users of this change.

File: .github/workflows/build.yml (hunk 1 above)
Comment (eladb) [reply .github/workflows/build.yml:2]: I want this workflow to run against any push (both PRs and pushes to master)

File: examples/java/crd/package.json (hunk 2 above)
Comment (eladb) [reply examples/java/crd/package.json:11]: It's much easier to manage this at the root level I believe. I'll also revisit the upgrade workflow shortly (via projen) after this first iteration is complete

**Comment by eladb** [comment]
@iliapolo Ready for another round

**Review by iliapolo — APPROVED** [review]
